tests/test.py

```diff
@@ -2176,6 +2176,7 @@ def testAcraServer(self):
             'acraserver_request_processing_seconds_bucket': {'min_value': 0},
 
             'acra_acrastruct_decryptions_total': {'min_value': 1},
+            'acra_decryptions_total': {'min_value': 1},
```

Comment:
will be great to add tests where all these metrics are used and we can validate them.  Make a request to the table that is configured for all types of tokenization, for all types of encryption and decryption and validate, that all appropriate metrics were incremented. And same for acra-translator and its API. As I see, previously it just called `HexFormatTest` test case which forced simple acrastruct decryption.  Now we have much more features) But to simplify tests, we can re-use some existing tests, imho.